src/aap_eda/services/project/scm.py

```diff
@@ -355,7 +360,4 @@ def is_refspec_valid(refspec: str, is_branch: bool) -> bool:
                 f"{result.returncode}"
             )
             return False
-        if is_branch and all(arg in refspec for arg in ["{{", "}}", "lookup"]):
```

Comment:
Is this removal related to this PR? it does not looks like.

Reply:
Git allows a branch that looks like this
"{{lookup('lines','id')}}" 
with no spaces. If there are spaces the git check-ref-format rejects it.

While we were waiting on the [unsafe](https://docs.ansible.com/ansible/latest/playbook_guide/playbooks_advanced_syntax.html) approach we need a temporary fix and now that we have unsafe strings we should allow users to create the branches with those names.